dpnp/backend/kernels/dpnp_krnl_mathematical.cpp

```diff
@@ -83,6 +83,25 @@ template void dpnp_elemwise_absolute_c<float>(void* array1_in, void* result1, si
 template void dpnp_elemwise_absolute_c<long>(void* array1_in, void* result1, size_t size);
 template void dpnp_elemwise_absolute_c<int>(void* array1_in, void* result1, size_t size);
 
+template <typename _KernelNameSpecialization1, typename _KernelNameSpecialization2, typename _KernelNameSpecialization3>
+class dpnp_cross_c_kernel;
+
+template <typename _DataType_input1, typename _DataType_input2, typename _DataType_output>
+void dpnp_cross_c(void* array1_in, void* array2_in, void* result1, size_t size)
+{
+    _DataType_input1* array1 = reinterpret_cast<_DataType_input1*>(array1_in);
+    _DataType_input2* array2 = reinterpret_cast<_DataType_input2*>(array2_in);
+    _DataType_output* result = reinterpret_cast<_DataType_output*>(result1);
+
+    result[0] = array1[1] * array2[2] - array1[2] * array2[1];
+
+    result[1] = array1[2] * array2[0] - array1[0] * array2[2];
+
+    result[2] = array1[0] * array2[1] - array1[1] * array2[0];
```

Comment:
Is it really common enough case for this function?
This looks like a very special case.